src/assets/i18n/fr.json

```diff
@@ -0,0 +1,153 @@
+    "password_input": "Mot de passe",
+    "delete_wallet_button": "Suppression du portefeuille",
+    "delete_confirmation": {
+      "title": "Confirmation de la suppression",
+      "description": "Etes-vous sûre de vouloir supprimer ce portefeuille ?",
+      "delete_button": "Supprimer",
+      "cancel_button": "Annuler"
+    },
+    "delete_outcome": {
+      "success": "Portefeuille supprimer",
+      "success_description": "Le portefeuille sélectionné a bien été supprimer !",
+      "success_button": "OK",
+      "not_found": "Portefeuille inexistant",
+      "not_found_description": "Ce portefeuille n'existe pas. Veuillez redémarrer votre application.",
+      "not_found_button": "OK",
+      "incorrect_password": "Mot de passe incorrect !",
+      "incorrect_password_description": "Saisi de mot de passe incorrect, veuillez le ressaisir.",
+      "incorrect_password_button": "OK"
+    }
+  },
+  "generate_wallet_modal": {
+    "title": "Créer et sauver",
+    "close": "Fermer",
+    "address_input": "Adresse",
+    "private_key_input": "Clé privée",
+    "private_key_generate_button": "Créer",
+    "wallet_name_input": "Nom du portefeuille",
+    "password_input": "Mot de passe",
+    "confirm_password_input": "Confirmation du mot de passe",
+    "save_wallet_button": "Sauvegarde du portefeuille"
+  },
+  "import_wallet_modal": {
+    "title": "Importation de portefeuille",
+    "close": "Fermer",
+    "private_key_input": "Clé privée",
+    "wallet_name_input": "Nom du portefeuille",
+    "password_input": "Mot de passe",
+    "confirm_password_input": "Confirmation du mot de passe",
+    "save_wallet_button": "Sauvegarde du portefeuille",
+    "import_outcome": {
+      "invalid": "Clé privée invalide !",
+      "invalid_description": "Impossible de générer une adresse en utilisant cette clé !",
+      "invalid_button": "OK"
+    }
+  },
+  "load_wallet_modal": {
+    "title": "Déchiffrement et Chargement",
+    "close": "Fermer",
+    "wallet_name_input": "Nom du portefeuille",
+    "password_input": "Mot de passe",
+    "load_wallet_button": "Chargement du portefeuille",
+    "load_outcome": {
+      "incorrect_password": "Mot de passe incorrect !",
+      "incorrect_password_description": "Saisi de mot de passe incorrect, veuillez le ressaisir.",
+      "incorrect_password_button": "OK"
+    }
+  },
+  "save_wallet_modal": {
+    "title": "Chiffrer et Sauver",
+    "close": "Fermer",
+    "address_input": "Adresse",
+    "private_key_input": "Clé privée",
+    "wallet_name_input": "Nom du portefeuille",
+    "password_input": "Mot de passe",
+    "confirm_password_input": "Confirmation du mot de passe",
+    "save_wallet_button": "Sauver"
+  },
+  "view_transaction_modal": {
+    "title": "Détail de la transaction",
+    "close": "Fermer",
+    "overview_card_title": "Aperçu",
+    "txn_id": "Indentifiant",
+    "hash": "Block Hash",
+    "index": "Block Index",
+    "amount": "Montant",
```

Comment:
_Montant de la transaction_

Reply:
I thought it would be a bit heavy on the list head line, any opinion ?